components/Tabs/src/MDCTabBarViewController.m

```diff
@@ -192,17 +200,43 @@ - (nullable UIViewController *)controllerWithView:(nullable UIView *)view {
   return nil;
 }
 
+- (void)viewSafeAreaInsetsDidChange {
+  // TODO(rsmoore): Remove check when we drop Xcode 7/8 support
+#if defined(__IPHONE_11_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_11_0)
+  if (@available(iOS 11.0, *)) {
+    [super viewSafeAreaInsetsDidChange];
+  }
+#endif
+  [self updateLayout];
```

Comment:
Do we want to updateLayout or setNeedsLayout?
Does [UIView viewSafeAreaInsetsDidChange] not already call setNeedsLayout?

Reply:
`setNeedsLayout` makes sense. This gets called just before `viewDidLayoutSubviews` when we load/rotate.